program/program/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+namespace  ConsoleApp
+{
+    class Program
+    {
+        static void Main()
+        {           
+            //first task
+            Console.WriteLine("First task:");
+            Console.WriteLine($"F(n)={ sumF(1, 0, 5)}");
+            Console.WriteLine($"M(n)={ sumM(1, 0, 8)}");
+
+            //second task
+
+            Console.WriteLine("Second task:");
+            Console.WriteLine(Pow(4, 3));
+
+
+            //third task
+            
+            Console.WriteLine($"Count number:");
+            Count(1,10);
+
+
+        }
+        //first task
+        static int sumF(int F, int M, int n)
```

Comment:
it something incorrect
function in hometask was recursive, will explain in more details during week